packages/playground-ui/src/components/ui/elements/entry-list/entry-list.tsx

```diff
@@ -1,15 +1,17 @@
 import { ArrowLeftIcon, ArrowRightIcon } from 'lucide-react';
-import { EntryListCell } from './entry-list-cell';
+import { EntryListTextCell } from './entry-list-cell';
 import { EntryListItem } from './entry-list-item';
 import { getColumnTemplate, type Column } from './shared';
 
 import { cn } from '@/lib/utils';
+import { isValidElement } from 'react';
 
 export function EntryList({
-  items,
-  selectedItem,
+  items: dataItems,
```

Comment:
I don't think we need this. We could probably keep EntryList as a UI component (with no information about any entities type or whatever.

What we could do instead is
- reducing the EntryList props list
- add a {children} instead
- create a "business component" that knows about domain types, that uses EntryList and all the other things

So that we can have a generic EntryList.

Reply:
will think of it in the next iteration